src/AppPrefs.cpp

```diff
@@ -240,10 +240,10 @@ void UpdateGlobalPrefs(const CommandLineInfo& i) {
                    str::EqI(i.globalPrefArgs.At(n), L"-bg-color")) {
             // -bgcolor is for backwards compat (was used pre-1.3)
             // -bg-color is for consistency
-            ParseColor(&gGlobalPrefs->mainWindowBackground, i.globalPrefArgs.At(++n));
+            //ParseColor(&gGlobalPrefs->mainWindowBackground, i.globalPrefArgs.At(++n));

```

Comment:
This seems rather messy, and may be part of a larger backwards compatibility problem.